src/DataflowType/Dataflow/Dataflow.php

```diff
@@ -94,4 +106,13 @@ private function processItem($item): void
             $writer->write($item);
         }
     }
+
+    private function logException(\Throwable $e, ?string $index = null): void
+    {
+        if (!isset($this->logger)) {
+            return;
+        }
+
+        $this->logger->error($e, ['index' => $index]);
```

Comment:
The exception key in the context array permits the log of many to debug information (like the line and the file throw the exception)

```suggestion
        $this->logger->error($e, ['exception'=>$e, 'index' => $index]);
```